src/auth/device-auth-flow.ts

```diff
@@ -207,6 +207,32 @@ export async function refreshAccessToken(selectedScopes?: string[]): Promise<str
   }
 }
 
+export async function revokeRefreshToken() {
```

Comment:
I think it would be useful to add proper JSDoc documentation explaining this is for future use when refresh tokens are supported.